Makefile

```diff
@@ -1,4 +1,16 @@
-########################################
+# ------------------------ User commands -----------------------
+# gnokey        Key manipulation, also general interaction with gnoland
+# gnoland       Run the blockchain node
```

Comment:
I like the idea, I would just use the present continuous tense for the description verbs:
`Runs`, `Serves`, `Imports` etc